### ISSUE TITLE
Terms and conditions

### ISSUE THREAD

**Issue by govuk-design-system** [alphagov/govuk-design-system-backlog#102]
## What
Help users understand the terms and conditions of using your service.

## Why
Most government services have terms and conditions of use. The best services integrate this information into the service at the relevant points, rather than burying it all in a long page that users won't read.

## Anything else
- [Terms and conditions in HMRC design patterns](https://github.com/hmrc/design-patterns/issues/121)

**Comment by mikeash82** [comment]
@olirose commented on 3 Oct 2017

Here are the 'terms of participation' from the MTDfB sign up journey for both individuals and agents. These were signed off by the Deputy Director for MTDfB (policy and legal). 

Individual
![image](https://user-images.githubusercontent.com/31846170/31123261-0e6e007e-a837-11e7-8733-f10bab9c5935.png)

Agent
![image](https://user-images.githubusercontent.com/31846170/31123344-6fe49b60-a837-11e7-9f77-1d5117d97219.png)

**Comment by soupdragon99** [comment]
# Dropbox Paper audit

On 13th March 2019 the Design System team reviewed a Dropbox Paper document discussing the terms and conditions pattern.

The aim was to reduce the number of places containing guidance and code by:

- migrating relevant, useful content into the Design System itself
- recording important research findings in the community backlog
- removing the original Dropbox Paper page

If you need to, you can [see the original Dropbox Paper content in the internet archive](https://web.archive.org/web/20190313091120/https://paper.dropbox.com/doc/Terms-and-conditions-pages--AZMIWZKnVpOFk_hC5e1duFR0Ag-3mDeXlG9FZjeFMz4dUeQr).

## Review outcomes

Combine the terms and conditions page discussion on Dropbox Paper with this issue and remove the original Dropbox Paper page.

# Terms and conditions pages

## Overview

Help your users understand any important terms and conditions relating to their use of your service.

## When to use this pattern

Try to avoid the need for this pattern by keeping your service simple and incorporating any important conditions of use into the relevant parts of the service itself.

Your service may already be adequately covered by the [GOV.UK terms and conditions](https://www.gov.uk/help/terms-conditions) and [privacy policy](https://www.gov.uk/help/privacy-policy).

If you still need to create your own terms and conditions then follow the guidance below.

## How it works

### Make your terms and conditions easy to read and access

- keep your terms and conditions as concise as possible
- write your terms and conditions in plain English
- publish your terms and conditions so people can access them without having to sign in
- notify existing users if the terms and conditions of your service change
- make dated versions of all the previous terms and conditions available

### Repeat the most important points in the relevant parts of the service

Don’t expect your users to read and accept all the terms and conditions before using your service.
Instead, explain any important terms and conditions at the relevant point in the service.›

For example, in the ‘Register as a waste carrier' service some of the user data goes into a public register. This is important and not what users would expect, so they are told that this is happening at the relevant point.

### Get the appropriate consent from your users 

Establish whether you require implicit or explicit consent from your users. This will vary depending on the service and the legal opinion. [Common law rule](https://en.wikipedia.org/wiki/J_Spurling_Ltd_v_Bradshaw) states that the more extreme the term you're asking someone to agree to, the more you have to draw it to their attention.

### Options for getting consent

Here are some ways to obtain implicit or explicit consent:

**Implicit consent at start of service**
By using this service you accept the [terms and conditions]()

**Implicit consent at end of service**
By submitting your claim you accept the [terms and conditions]()

**Explicit consent, button**
To use this service you must accept the [terms and conditions]()
[   I accept   ]

**Explicit consent, checkbox**
[x] ~I accept the terms and conditions~

![terms-and-conditions](https://user-images.githubusercontent.com/23702161/54268884-1efe0e80-4574-11e9-858c-8c306b02eae4.png)

**Embedded terms and conditions**

## Research relating to this pattern
### HMRC

In user research for the HMRC exemplar services users consistently either skipped over the T&Cs links or if they did click they commented that they wouldn't usually read T&Cs, that they probably should, or that they would if "[the service] was important”. 

## Related links
- https://en.wikipedia.org/wiki/J_Spurling_Ltd_v_Bradshaw

**Comment by frankieroberto** [comment]
2 examples of removing checkbox when agreeing to terms and conditions:

* [Submit cosmetic product notifications](https://github.com/OfficeForProductSafetyAndStandards/cosmetic-product-notifications/pull/1438)
* [Apply for teacher training](https://github.com/DFE-Digital/apply-for-teacher-training/pull/6835) – this was done after stats showed that 4% of users missed the checkbox and got an error

**Comment by CharlotteDowns** [comment]
We've just [added some guidance on how to 'share findings about your users' with us](https://design-system.service.gov.uk/community/share-research-findings/) 📝. This will help us learn more about how your users understand terms and conditions within your service.
